collectors/cochrane/parser.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from copy import deepcopy
+try:
+    from lxml import etree
+except ImportError:
+    import xml.etree.ElementTree as etree
+from .record import Record
+
+
+def parse_record(url, file_path):
+    tree = etree.parse(file_path)
+    study_robs = []
+    studies = []
+
+    # Get risk of bias
+
+    root = tree.getroot()
+    doi_id = root.attrib.get('DOI', '')
+    quality_item_data_entries = tree.findall('//QUALITY_ITEM_DATA_ENTRY')
+    for quality_item_data_entry in quality_item_data_entries:
+        study_rob = {}
```

Comment:
Same as the comment below, prefer to do:

```python
study_rob = {
    'study_id': 'foo',
    'modified': 'bar',
    # ...
}
```

Unless you can't (e.g. when you're doing some loops and more complicated stuff)